### ISSUE TITLE
PMI Usage message is incorrect

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/jitutils#140]
The dotnet CLI-version of the usage message prints "dotnet.exe" as the tool, which is confusing:

```
c:\gh\jitutils>where pmi
c:\gh\jitutils\bin\pmi.bat

c:\gh\jitutils>pmi -?
Usage:

  dotnet.exe Count PATH_TO_ASSEMBLY
      Count the number of types and methods in an assembly.

  dotnet.exe PrepOne PATH_TO_ASSEMBLY INDEX_OF_TARGET_METHOD
      JIT a single method, specified by a method number.

  dotnet.exe PrepAll PATH_TO_ASSEMBLY [INDEX_OF_FIRST_METHOD_TO_PROCESS]
      JIT all the methods in an assembly. If INDEX_OF_FIRST_METHOD_TO_PROCESS is specified, it is the first
      method compiled, followed by all subsequent methods.

  dotnet.exe DriveAll PATH_TO_ASSEMBLY
      The same as PrepAll, but is more robust. While PrepAll will stop at the first JIT assert, DriveAll will
      continue by skipping that method.

Environment variable PMIPATH is a semicolon-separated list of paths used to find dependent assemblies.

For Prepall and Prepone, optional suffixes will change output behavior:
   -Quiet will suppress in-progress messages for type and method exploration
   -Time will always show elapsed times, even in -Quiet mode
```

This is from:
```
private static int Usage()
{
    string exeName = System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName; // get the current full path name of PMI.exe
    exeName = System.IO.Path.GetFileName(exeName); // strip off the path; just use the EXE name.
    Console.WriteLine(
        "Usage:\r\n"
        + "\r\n"
        + "  " + exeName + " Count PATH_TO_ASSEMBLY\r\n"
```

**Comment by BruceForstall** [comment]
@AndyAyersMS

**Comment by AndyAyersMS** [comment]
The exe name is indeed dotnet.exe. There is no implicit CLR bootstrapper for core apps like there is for full framework apps.

IIRC there is some way to publish so you end up with a suitably named bootstrapping exe (perhaps by publishing for specific runtimes?). So maybe we should switch to that.

**Comment by BruceForstall** [comment]
Even if the name is dotnet.exe, the command line needs to include the pmi.dll.

**Comment by kunalspathak** [comment]
Looks like this is fixed in #144.
